Tests/Invoke-DockerPull.Tests.ps1

```diff
@@ -0,0 +1,56 @@
+Import-Module $PSScriptRoot/../Docker.Build.psm1
```

Comment:
You should add tests that cover the scenario where you get an invalid digest and throw an exception.

Reply:
Done.